docs/quickstart.rst

```diff
@@ -50,55 +50,90 @@ Create a Python file in the ``${RV_QUICKSTART_CODE_DIR}`` named ``tiny_spacenet.
            train_label_uri = '{}/buildings_AOI_2_Vegas_img205.geojson'.format(base_uri)
            val_image_uri = '{}/RGB-PanSharpen_AOI_2_Vegas_img25.tif'.format(base_uri)
            val_label_uri = '{}/buildings_AOI_2_Vegas_img25.geojson'.format(base_uri)
+           channel_order = [0, 1, 2]
 
-           task = rv.TaskConfig.builder(rv.OBJECT_DETECTION) \
-                               .with_chip_size(300) \
+           # ------------- TASK -------------
```

Comment:
In the paragraph below the code there is a reference to object detection. This should be changed to semantic segmentation.

Reply:
Done